README.md

```diff
@@ -48,7 +48,10 @@ No. We continue to consume Fluent Bit from its main repository. We are not forki
 
 ### Public Images
 
-Each release updates the `latest` tag and adds a tag for the version of the image.
+Each release updates the `latest` tag and adds a tag for the version of the image. A `stable` tag is also available which can be trusted that it is the latest version in which there are no high impact bugs. A release may be marked as stable if the following rules are all met:
+* It has been out for a period of time
+* The Fluent Bit version contained in the release is a patch version as the patch version indicates the release includes bug fixes
```

Comment:
Let's remove this criteria. We will use it internally but let's not say it publicly. Because if we say it publicly we are admitting we don't trust the fluent bit upstream to release stable 1.x.0 versions.